onnxmltools/convert/common/_topology.py

```diff
@@ -81,10 +82,17 @@ def input_full_names(self):
     @property
     def output_full_names(self):
         '''
-        Return all outpu variables' names
+        Return all output variables' names
         '''
         return [variable.full_name for variable in self.outputs]
 
+    @property
+    def original_operator(self):
```

Comment:
We directly access raw_operator in many places, so you need to go through the code base and change those `raw_operator`s to `original_operator`s as well. On the other hand, I don't exactly know why we need `original_operator`. Maybe it's a just Python convention for defining a class and not related to customized operator? If so, can we use another PR for revising the class hierarchy of onnxmltools?

Reply:
This original_operator is defined in the interface so that the end user can use it publicly.

The internal code should be ok with both raw_pointer and original_pointer since there is no way to assign a real property to the abstract property method with the same name.